src/extension/injected-script/addEventListener.ts

```diff
@@ -53,7 +53,8 @@ import type { CustomEvents } from './CustomEvents'
                 document.activeElement!.dispatchEvent(e)
                 return undefined!
             } else if (textOrImage.type === 'image') {
-                const xray_binary = globalThis.window.Uint8Array.from(textOrImage.value)
+                const Uint8Array = globalThis.Uint8Array ? globalThis.Uint8Array : globalThis.window.Uint8Array
```

Comment:
There's a `globalThis.Uint8Array` in firefox. `globalThis.window.Uint8Array` is undifined in firefox

Reply:
🤔🤔🤔🤔🤔🤔🤔🤔🤔